src/validators/CoinbaseSmartWalletValidator.sol

```diff
@@ -12,10 +12,21 @@ contract CoinbaseSmartWalletValidator is IAccountStateValidator {
     /// @notice Error thrown when an account has no nextOwnerIndex
     error Unintialized();
 
+    /// @notice The implementation of the CoinbaseSmartWallet this validator expects
+    CoinbaseSmartWallet public immutable walletImplementation;
```

Comment:
thoughts on renaming to `supportedImplementation` for more precision?

Reply:
love it